main/src/rootbrowse.cxx

```diff
@@ -0,0 +1,130 @@
+// \file rootbrowse.cxx
+///
+/// Command line tool to open a ROOT file on a TBrowser
+///
+/// \author Giacomo Parolini <giacomo.parolini@cern.ch>
+/// \date 2025-08-21
+#include <cstring>
+#include <iostream>
+#include <thread>
+#include <chrono>
+#include <memory>
+
+#include <ROOT/RLogger.hxx>
+
+#include <TError.h>
+#include <TFile.h>
+#include <TROOT.h>
+#include <TSystem.h>
+#include <TBrowser.h>
+#include <TApplication.h>
+
+static const char *const kLongHelp = R"(
+Open a ROOT file in a TBrowser
+
+positional arguments:
+  FILE           Input file
+
+options:
+  -h, --help     show this help message and exit
+  -w, --web WEB  Configure webdisplay. For all possible values, see:
+                 https://root.cern/doc/v636/classTROOT.html#a1749472696545b76a6b8e79769e7e773
```

Comment:
Not sure if there is a way to point to the latest doc consistently...

Reply:
That would be great! Or this string needs to be updated as part of the release procedure

Reply:
Why not doing it on the c++ side or even CMake side? since R__VERSION_MAJOR is known, we can decide whether the help prints:
v636 or v634 or master, etc.
For non-even dev versions, either "master" or round-up to even number.
This could be a generic CMake or C++ function that would be useful for everybody, not just rootbrowse.

Reply:
I still don't hate the idea of having a `latest-stable` link, regardless of what we decide to do with this PR